src/solvers/smt/smt_conv.h

```diff
@@ -330,6 +330,20 @@ class smt_convt
   virtual smt_astt mk_real2int(smt_astt a);
   virtual smt_astt mk_int2real(smt_astt a);
   virtual smt_astt mk_isint(smt_astt a);
+  virtual smt_astt
+  mk_quantifier(bool is_forall, std::vector<smt_astt> lhs, smt_astt rhs)
+  {
+    (void)is_forall;
+    (void)lhs;
+    (void)rhs;
+    abort();
+  }
+  virtual smt_astt mk_exists(smt_astt symbols, smt_astt body)
```

Comment:
Did I miss where this method is used?

Reply:
Nice catch! Missed it while refactoring.